### DISCUSSION TITLE
I guess I know why protonmail can't auto-reply the email from simplelogin

### DISCUSSION THREAD

**Discussion by JasonMoriarty** [simple-login/app#1239] [Q&A]
It can not reply if the recipient address is not a part of the user's addresses.

I don't know if any change can do with simplelogin on this setting. Or it has to be done on ProtonMail.

**Comment by nguyenkims** [comment]
@JasonMoriarty thanks for reporting it, we are investigating and will post updates as soon as we have more info.
